docs: update readmes (#3315)
digdir/designsystemet: packages/css/README.md

CSS implementation of the Designsystemet components

- Uses [@layer](https://developer.mozilla.org/en-US/docs/Web/CSS/@layer) `ds`

> - Uses `@digdir/designsystemet-theme` or CSS file generated from `@digdir/designsystemet tokens build` for theming,

Reviewer: ```suggestion
- Uses `@digdir/designsystemet-theme` or CSS file generated from `@digdir/designsystemet tokens build` for theming
```

  - Build your own theme on https://theme.designsystemet.no/

Read the Designsystemet [README](https://github.com/digdir/designsystemet) to get started.

## Q&A

### Tailwind

> If you are using Tailwind with (Preflight)[https://tailwindcss.com/docs/preflight] make sure to load it before Designsystemet.

Reviewer: ```suggestion
If you are using Tailwind with (Preflight)[https://tailwindcss.com/docs/preflight] make sure to load Designsystemet first.
```
